spacer/tasks.py

```diff
@@ -28,8 +27,14 @@ def extract_features(msg: ExtractFeaturesMsg) -> ExtractFeaturesReturnMsg:
 
     check_extract_inputs(img, msg.rowcols, msg.image_loc.key)
 
+    check_rowcols(msg.rowcols, img)
+    if not isinstance(msg.extractor, Callable):
+        raise TypeError("msg.extractor must be callable")
+    
     with config.log_entry_and_exit('actual extraction'):
         features, return_msg = msg.extractor(img, msg.rowcols)
+    if not hasattr(features, 'store'):
+        raise TypeError("features object must have a store method")
```

Comment:
These two type checks (well, instance/attribute checks to be exact) aren't that related to the rest of the PR.

They're also not really needed in my opinion - I think in Python, type checks don't hurt individually but they can add up to clutter the codebase somewhat. I know Oscar had several such checks in the codebase here and there, but in general I wasn't really keen on adding more. Sometimes they're a good idea with recent API changes (like PR #62 which admittedly relates to the first check here), but I'd like to think that's more of a worry for apps with larger userbases than pyspacer's.